### ISSUE TITLE
DietPi-Software | Cloudprint (CUPS) web interface not functional

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#1797]
> https://dietpi.com/phpbb/viewtopic.php?f=11&t=3781&p=12736#p12733

**Comment by Fourdee** [comment]
Notes:

- during install
```
cupsctl: Unable to connect to server: Bad file descriptor
```

Due to service not running.

**Comment by Fourdee** [comment]
@oparoz 

Resolved for v6.9, please run the following commands to resolve your current install of cups:
```
cupsctl --remote-admin
usermod -a -G lpadmin root
dietpi-services restart
```
